packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2417,13 +2417,20 @@ is required to delete the user.
 | id        | `id` of user                                                                          | URL        | String   | TRUE     |         |
 | sendEmail | Sends a deactivation email to the administrator if `true`.  Default value is `false`. | Query      | Boolean  | FALSE    | FALSE   |
 
+> User deletion can also be performed asynchronously. To invoke asynchronous user deletion, pass an HTTP header
+> `Prefer: respond-async` with the request. This header is also supported by user deactivation, which will
+> be performed asynchronously if the delete endpoint is invoked on a user that has not been deactivated.
+
 ##### Response Parameters
 
 
 `204 No Content`
 
 Passing an invalid `id` returns a `404 Not Found` status code with error code `E0000007`.
 
+#### Delete user
```

Comment:
I noticed that we have a header "Delete User" at L2402 which is the header of the entire delete operation. Whereas this header is the header of the synchronous delete user request example. Would it make sense to change this header to "Delete user synchronously" to match with the "Delete user asynchronously" header a few lines below?

Reply:
Updated.